target/linux/layerscape/image/armv8_64b.mk

```diff
@@ -222,12 +234,13 @@ endef
 TARGET_DEVICES += ls2088ardb
 
 define Device/traverse-ls1043
+  DEVICE_VENDOR := Traverse
+  DEVICE_MODEL := LS1043
```

Comment:
I would keep the "LS1043 Boards" here, it sounds like a development board/platform (unless you now better).